### PR TITLE
Manual Update : For Edge Setup 

### PR DESCRIPTION
**Pull request by pradumnnec**
Manual is updated in accordance to the latest code flow and structure.

### CODE REVIEW

File: docker/edge/http/edge_stop.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+docker stop edgebroker && docker rm $_
+docker stop edgeworker && docker rm $_
```
Comment (naveennec) [inline docker/edge/http/edge_stop.sh:2]: do newline is not needed at the end like in above script edge_start.sh is having.

File: docker/edge/http/edge_start.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+if [ $# -eq 0 ]; then
```
Comment (naveennec) [inline docker/edge/http/edge_start.sh:1]: mention in the start of the script  as a comment what should be input parameter and what this script is designed for.

File: docker/edge/http/edge_stop.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+docker stop edgebroker && docker rm $_
```
Comment (naveennec) [inline docker/edge/http/edge_stop.sh:1]: mention in the start of the script what should be input prameter and what this script is designed for.

**Review by naveennec — CHANGES_REQUESTED** [review]

File: docker/edge/http/edge_stop.sh (hunk 1 above)
Comment (pradumnnec) [reply docker/edge/http/edge_stop.sh]: Comment Incorporated

File: docker/edge/http/edge_start.sh (hunk 2 above)
Comment (pradumnnec) [reply docker/edge/http/edge_start.sh:7]: Comment Incorporated

File: docker/edge/http/edge_stop.sh (hunk 3 above)
Comment (pradumnnec) [reply docker/edge/http/edge_stop.sh:3]: Comment Incorporated

**Review by naveennec — APPROVED** [review]

**Review by naveennec — APPROVED** [review]

**Pull request merged**